tests/foreman/cli/test_oscap.py

```diff
@@ -77,21 +62,20 @@ def fetch_scap_and_profile_id(cls, scap_name, scap_profile):
         ]
         return scap_id, scap_profile_ids
 
-    @classmethod
-    def setUpClass(cls):
-        super(OpenScapTestCase, cls).setUpClass()
-        cls.title = gen_string('alpha')
-        result = [scap['title'] for scap in Scapcontent.list() if scap.get('title') in cls.title]
+    @pytest.fixture(scope="class")
+    def scap_content(self, import_ansible_roles, import_puppet_classes):
```

Comment:
This fixture can be moved out of this class, So other classes in this module now/later can use them. The scope can be class though.

Reply:
moved to `oscap_fixtures.py`

Reply:
@jameerpathan111 Cherry on the cake :)